lib/manageiq/rpm_build/generate_tar_files.rb

```diff
@@ -19,50 +13,37 @@ def shell_cmd(cmd)
 
       def create_gemset_tarball
         puts "\n---> #{self.class.name}::#{__method__}"
-        Dir.chdir(BUILD_DIR) do
-          gemset_public_dir = gem_home.join("vmdb/public")
-          FileUtils.mkdir_p(gemset_public_dir)
-          # Can't be symlink, as files need to be tar'ed without '-h' to keep symlink for git based gems
-          FileUtils.cp_r(BUILD_DIR.join("manageiq/public/assets"), gemset_public_dir)
-          FileUtils.cp_r(BUILD_DIR.join("manageiq/public/packs"), gemset_public_dir)
 
-          # Override path in bundler plugin index
-          plugin_index = gem_home.join("vmdb/.bundle/plugin/index")
-          plugin_index.write(plugin_index.read.gsub(BUILD_DIR.join("manageiq").to_s, '/var/www/miq/vmdb'))
+        tar_basename = "#{PRODUCT_NAME}-gemset-#{VERSION}"
+        tar_name = RPM_SPEC_DIR.join("#{tar_basename}.tar.gz")
+
+        # Override path in bundler plugin index
+        plugin_index = GEM_HOME.join("vmdb/.bundle/plugin/index")
+        plugin_index.write(plugin_index.read.gsub(BUILD_DIR.join("manageiq").to_s, '/var/www/miq/vmdb'))
 
-          shell_cmd("tar -zcf #{PRODUCT_NAME}-gemset-#{VERSION}.tar.gz #{PRODUCT_NAME}-gemset-#{VERSION}/")
-        end
-        FileUtils.cp(BUILD_DIR.join("#{PRODUCT_NAME}-gemset-#{VERSION}.tar.gz"), RPM_SPEC_DIR.join("#{PRODUCT_NAME}-gemset"))
+        shell_cmd("tar -C #{BUILD_DIR} -zcf #{tar_name} #{tar_basename}")
       end
 
       def create_appliance_tarball
         puts "\n---> #{self.class.name}::#{__method__}"
-        Dir.chdir(BUILD_DIR) do
-          transform = "--transform s',\^,#{PRODUCT_NAME}-appliance-#{VERSION}\/,\'"
-          base_dir = BUILD_DIR.join("manageiq-appliance")
-          shell_cmd("tar -C #{base_dir} #{transform} --exclude='.git' -hzcf #{PRODUCT_NAME}-appliance-#{VERSION}.tar.gz .")
 
-          FileUtils.cp(BUILD_DIR.join("#{PRODUCT_NAME}-appliance-#{VERSION}.tar.gz"), RPM_SPEC_DIR.join("#{PRODUCT_NAME}-appliance"))
-        end
+        tar_basename = "#{PRODUCT_NAME}-appliance-#{VERSION}"
+        tar_name  = RPM_SPEC_DIR.join("#{tar_basename}.tar.gz")
+        transform = "--transform s',\^,#{tar_basename}\/,\'"
```

Comment:
I code elsewhere I thought we had a check for Mac vs other here?

Reply:
This was the only place it was checking, so I just removed it... especially now with a container image that can run the build, I thought it's not needed. I never tried running on Mac, so I'm not sure if transform is the only thing that needs the check, but I can add if needed.

Reply:
oh interesting...if I recall some part of the `--transform` thing was added because on a Mac the tar command works slightly differently.

Reply:
So, yeah, now that we are container based, we can ignore Mac specifics.